package.json

```diff
@@ -93,21 +93,21 @@
     "style-loader": "^1.2.1",
     "webpack": "^4.43.0",
     "webpack-cli": "^3.3.11",
-    "webpack-dev-server": "^3.11.0"
+    "webpack-dev-server": "^3.11.0",
+    "@emotion/react": "^11.4.1",
+    "@emotion/styled": "^11.3.0"
   },
   "peerDependencies": {
     "@mui/icons-material": "^5.0.1",
     "@mui/material": "^5.0.1",
     "react": "^16.8.0 || ^17.0.2",
-    "react-dom": "^16.8.0 || ^17.0.2",
-    "tss-react": "^3.4.1"
+    "react-dom": "^16.8.0 || ^17.0.2"
   },
   "dependencies": {
     "@babel/runtime-corejs3": "^7.12.1",
-    "@emotion/react": "^11.4.1",
-    "@emotion/styled": "^11.3.0",
     "@emotion/cache": "^11.7.1",
-    "tss-react": "^3.4.1",
+    "tss-react": "^3.5.2",
```

Comment:
`tss-react` updated to a version that doesn’t require for it to be in `peerDependencies`